.gitignore

```diff
@@ -3,3 +3,4 @@ node_modules
 .DS_Store
 yarn.lock
 package-lock.json
+dist
```

Comment:
@ivadimko I remember you said we can't add dist to gitignore. Do you remember the reason?

Reply:
With `dist` under `.gitignore` we can't deploy it to GH pages. Our deploy script simply push `dist` folder so it should be committed.

Reply:
I solve this problem whit plugin `gh-pages`

Reply:
@ivadimko I guess in that case we can move forward.